LeaderboardBackend/Program.cs

```diff
@@ -19,12 +19,15 @@
 #region WebApplicationBuilder
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 
-DotEnv.Load(options: new DotEnvOptions(
-	ignoreExceptions: false,
-	overwriteExistingVars: false,
-	envFilePaths: new[] { builder.Configuration["EnvPath"] },
-	trimValues: true
-));
+if (!builder.Environment.IsProduction())
```

Comment:
I've merged https://github.com/leaderboardsgg/leaderboard-backend/pull/134, which doesn't use `DotEnv` anymore. This statement needs updating